test/Model/loading.js

```diff
@@ -2,19 +2,19 @@ var expect = require('../util').expect;
 var racer = require('../../lib/index');
 
 describe('loading', function() {
-  describe('subscribe', function() {
-    beforeEach(function(done) {
-      this.backend = racer.createBackend();
-      // Add a delay on all messages to help catch race issues
-      var delay = 5;
-      this.backend.use('receive', function(request, next) {
-        delay++;
-        setTimeout(next, delay);
-      });
-      this.model = this.backend.createModel();
-      this.model.connection.on('connected', done);
+  beforeEach(function(done) {
+    this.backend = racer.createBackend();
+    // Add a delay on all messages to help catch race issues
+    var delay = 5;
+    this.backend.use('receive', function(request, next) {
+      delay++;
```

Comment:
Does this mean we are increasing the delay for every `receive`?

Reply:
Seems like it. Nate wrote this originally. I think the intent is, if multiple messages are received in very quick succession on the server, to stagger them out a bit.